pyi.py

```diff
@@ -929,10 +929,15 @@ def visit_AnnAssign(self, node: ast.AnnAssign) -> None:
         if _is_TypeAlias(node_annotation):
             with self.visiting_TypeAlias.enabled():
                 self.generic_visit(node)
-            if isinstance(node_target, ast.Name) and self._Y043_REGEX.match(
-                target_name
-            ):
-                self.error(node, Y043)
+            if isinstance(node_target, ast.Name):
+                if target_name.startswith("_"):
+                    if target_name[1].islower():
```

Comment:
This will fail if you use `_` as a type alias. Maybe a regex would be better here? `if re.match(r"_?[a-z]", target_name)`

Reply:
Hah, nice spot! I'm busy this weekend but will make the change when I get the chance.